woocommerce/woocommerce
# CYS: improve CTA

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR addresses the feedback regarding the call-to-action (CTA) for saving changes.

| Before | After |
|--------|--------|
|![image](https://github.com/user-attachments/assets/a9821380-f625-4972-ad0c-9530a3d84c85) | ![image](https://github.com/user-attachments/assets/8729832b-e69d-4631-9266-78bd7023b21c) | 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Head over to WooCommerce > Home > Customize your store.
2. Follow the flow.
3. Ensure that the button on the left sidebar contains `Finish customizing` label




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: improve CTA

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
